src/gossip/fuzz_table.zig

```diff
@@ -94,8 +87,15 @@ pub fn run(seed: u64, args: *std.process.ArgIterator) !void {
                 defer put_count += 1;
                 const new_keypair = random.boolean();
 
-                var data: GossipData = .{
-                    .ContactInfo = try ContactInfo.initRandom(allocator, random, Pubkey.initRandom(random), 0, 0, 0),
+                var data = GossipData{
+                    .ContactInfo = try ContactInfo.initRandom(
+                        allocator,
+                        random,
+                        Pubkey.initRandom(random),
+                        0,
+                        0,
+                        0,
+                    ),
```

Comment:
not important, but fyi you can take this approach to stay under the line limit. personally I prefer this way.
```suggestion
                    .ContactInfo = try ContactInfo
                        .initRandom(allocator, random, Pubkey.initRandom(random), 0, 0, 0),
```